### PR TITLE
Update sentence_translations.json

### PR DESCRIPTION
**Pull request by cheskoxd**
-Fix broken sentences due to translation 
-Changed some sentences to sound more native
-Swapped some words with their common synonym
-Fix channel name broken due to translation 

Quick question,
Should we translate the name of the channel?

### CODE REVIEW

**Comment by imlargo** [comment]
¡Hola! También estoy trabajando en traducir al español. Con respecto a tu pregunta, creo que sería más adecuado dejar el nombre del canal en su idioma original, para mantener la idea y la "_marca personal_". Pero en todo caso, es simplemente mi opinión.

---

Hi! I'm also working on translating to Spanish. Regarding your question, I think it would be more appropriate to leave the channel name in its original language, in order to maintain the idea and the "_personal brand_". But in any case, that's just my opinion.

**Comment by cheskoxd** [comment]
> ¡Hola! También estoy trabajando en traducir al español. Con respecto a tu pregunta, creo que sería más adecuado dejar el nombre del canal en su idioma original, para mantener la idea y la "_marca personal_". Pero en todo caso, es simplemente mi opinión.
> 
> Hi! I'm also working on translating to Spanish. Regarding your question, I think it would be more appropriate to leave the channel name in its original language, in order to maintain the idea and the "_personal brand_". But in any case, that's just my opinion.

Hey, pensaba lo mismo, ya que se escucha raro "tres azules y un marron", pero dijeron que usarian voces AI para las traducciones, y si es asi podria afectar a la hora de crear el audio en español, ya que 3blue1brown esta en ingles y el texto a audio solo puede hablar en un idioma
<hr/>
Hey, I also think that its better if we leave it in English, because in Spanish it sounds weird, but since they are going to use AI Voices to generate the audio, it might/will affect since the word 3blue1brown its in english and the text to speech can only talk in one language.

Here's how it might/will sound:
https://translate.google.com/?sl=es&tl=en&text=3blue1brown%20&op=translate

File: 2023/ego-and-math/spanish/sentence_translations.json [hunk 1]
```diff
@@ -559,7 +559,7 @@
  },
  {
   "input": "When I started making videos about math I was actually a senior here at Stanford when I very first began I was uploading them to a channel that I had given the very strange name of three blue and brown and at that time I cared a lot about the content seeming very different from traditional lessons out there. ",
-  "translatedText": "Cuando comencé a hacer videos sobre matemáticas, en realidad era estudiante de último año aquí en Stanford, cuando comencé, los estaba subiendo a un canal al que le había dado el nombre muy extraño de tres azules y marrones y en ese momento me preocupaba mucho por el El contenido parece muy diferente de las lecciones tradicionales que existen. ",
+  "translatedText": "Cuando comencé a hacer videos sobre matemáticas, en realidad era estudiante de último año aquí en Stanford, cuando comencé, los estaba subiendo a un canal al que le había dado el nombre muy extraño de tres azules y un marrón y en ese momento me importaba mucho por que el contenido fuera muy diferente de las lecciones tradicionales que existen. ",
```
Comment (Yag000) [inline 2023/ego-and-math/spanish/sentence_translations.json:562]:
```suggestion
  "translatedText": "Cuando comencé a hacer videos sobre matemáticas, en realidad era estudiante de último año aquí en Stanford, cuando comencé, los estaba subiendo a un canal al que le había dado el nombre muy extraño de tres azules y un marrón y en ese momento me importaba mucho que el contenido fuera muy diferente de las lecciones tradicionales que existen. ",
```

File: 2023/ego-and-math/spanish/sentence_translations.json [hunk 2]
```diff
@@ -622,7 +622,7 @@
  },
  {
   "input": "I did a series about linear algebra I did a series about calculus people asked if I could explain what is a Fourier transform or how do neural networks work and so I obliged. ",
-  "translatedText": "Hice una serie sobre álgebra lineal. Hice una serie sobre cálculo. La gente me preguntó si podía explicar qué es una transformada de Fourier o cómo funcionan las redes neuronales y acepté. ",
+  "translatedText": "Hice una serie sobre álgebra lineal. Hice una serie sobre cálculo. La gente me preguntó si podía explicar qué es una transformacion de Fourier o cómo funcionan las redes neuronales y acepté. ",
```
Comment (Yag000) [inline 2023/ego-and-math/spanish/sentence_translations.json:625]:
```suggestion
  "translatedText": "Hice una serie sobre álgebra lineal. Hice una serie sobre cálculo. La gente me preguntó si podía explicar qué es una transformada de Fourier o cómo funcionan las redes neuronales y acepté. ",
```
The correct term in Spanish is actually `transformada` even if it doesn't seem like it: https://es.wikipedia.org/wiki/Transformada_de_Fourier

**Review by Yag000 — COMMENTED** [review]
Just some minor tweaks.

**Comment by Yag000** [comment]
Regarding the translation of the channel name, I believe it's quite tricky in this case. Firstly, I think we should always keep '3blue1brown' as it is the actual name of the channel. However, the name was explicitly referenced as a weird name, and its meaning may not be clear to those who don't understand English. A potential solution could be to retain the English name but add a brief explanatory phrase, for instance: '...3blue1brown, tres azules y un marrón en español,...' where it was mention, l.562

**Comment by cheskoxd** [comment]
> Regarding the translation of the channel name, I believe it's quite tricky in this case. Firstly, I think we should always keep '3blue1brown' as it is the actual name of the channel. However, the name was explicitly referenced as a weird name, and its meaning may not be clear to those who don't understand English. A potential solution could be to retain the English name but add a brief explanatory phrase, for instance: '...3blue1brown, tres azules y un marrón en español,...' where it was mention, l.562

Got it, Ill update the file and submit it for review, thanks for this clarification!

**Review by cheskoxd — COMMENTED** [review]
-Fix broken sentences due to translation
-Changed some sentences to sound more native
-Swapped some words with their common synonym
-Fix channel name broken due to translation
-Update to maintain the channel name and its translation in Spanish

**Comment by Yag000** [comment]
>  -Update to maintain the channel name and its translation in Spanish

I think I misexplained myself. What I meant to say is that we should only keep '3blue1brown' in the translation but add an explanation only the first time the name is mentioned, in line 562. If not, I believe it won't feel natural.

**Comment by cheskoxd** [comment]
> I think I misexplained myself. What I meant to say is that we should only keep `3blue1brown` in the translation but add an explanation only the first time the name is mentioned, in line 562. If not, I believe it won't feel natural.

Thanks for the clarification, I just added commit [d908a9d](https://github.com/3b1b/captions/pull/77/commits/d908a9d2e54929de47ceaad3daa5e4cc71409341) to fix that, and now its ready to merge

>I would leave the channel name as-is, and if the text-to-speech result does something very >unnatural, we can address that on a further stage of the pipeline, e.g. programmatically >replace it with a translation of "this channel" before running the TTS, or something like that.

You're right, using "this channel" sounds a lot better than using the translation.

>Let me know when you feel this is okay to merge.

I just submitted it for review

**Review by Yag000 — APPROVED** [review]
Good job :)

**Pull request merged**